tasks/scheduler.go

```diff
@@ -57,6 +57,7 @@ func (dt *deliverTxTask) Increment() {
 
 // Scheduler processes tasks concurrently
 type Scheduler interface {
+	PrefillEstimates(ctx sdk.Context, metadatas []sdk.DeliverTxMetadata)
```

Comment:
2 cents: if this always must happen prior to ProcessAll, it may be worth just including the logic in ProcessAll to simplify the interaction.